app.js

```diff
@@ -198,6 +183,40 @@ router.post(
     }
 );
 
+router.post(
+    '/account/resendRecoveryLink',
+    checkAccountOwnership,
+    async ctx => {
+        const { accountId, seedPhrase, publicKey, method } = ctx.request.body;
+        const [account] = await models.Account.findOrCreate({ where: { accountId } });
+        let email, phoneNumber;
+
+        if (method.kind === 'email') {
```

Comment:
nit: seems like we can just make `sendRecoveryMessage` accept `method` directly instead of `phoneNumber` / `email`